local.css

```diff
@@ -274,7 +274,21 @@ aside h4 {
 	margin-inline-start: 1em;
 	}
 
-
+table.whitespace {
+	/* width: 300px; */
+	width: 100%;
+    border-spacing: 0;
+	margin: 1em 0;
+	}
+table.whitespace th,
+table.whitespace td {
```

Comment:
centering the check marks horizontally looks better, but that would require a bit more work on styles.

Reply:
Fixed.